src/routes/Search/SearchForm.jsx

```diff
@@ -3,21 +3,20 @@ import PropTypes from 'prop-types';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
 
 import {
-  OutlinedInput,
```

Comment:
Ignore this entire diff. I have a separate branch where this will be reconciled properly